src/compiler/resolutionCache.ts

```diff
@@ -54,9 +56,17 @@ namespace ts {
         refCount: number;
     }
 
+    /*@internal*/
+    export const maxNumberOfFilesToIterateForInvalidation = 256;
+
+    interface GetResolutionWithResolvedFileName<T extends ResolutionWithFailedLookupLocations = ResolutionWithFailedLookupLocations, R extends ResolutionWithResolvedFileName = ResolutionWithResolvedFileName> {

```

Comment:
An interface containing just a single call signature, with a type parameter for every parameter and the return type, doesn't seem much better than using a function type directly.